src/components/Pickers/Lookup/Lookup.js

```diff
@@ -91,18 +148,23 @@ class Lookup extends React.PureComponent {
 
 	onMenuOpen = async () => {
 		await this.setState({
-			menuIsOpen: true,
-			optionsCache: this.props.alwaysRefresh ? this.initialOptionsCache : this.state.optionsCache
+			menuIsOpen: true
 		});
-
+		if (this.props.alwaysRefresh) {
+			await this.setState({
+				optionsCache: this.initialOptionsCache,
+				controlOptions: [],
+				isTriedLoad: false
+			});
+		}
 		const { optionsCache } = this.state;
 
-		if (this.props.alwaysRefresh || !some(optionsCache[""].options)) {
+		if (this.props.alwaysRefresh || optionsCache[""].options.length <= 1) {
```

Comment:
it'll break existing logic because options should be loaded if there are no options in optionsCache or if selected one was added to the cache. if i remove <=1 it'll reverse previous condition to opposite.